src/gnssro/gnssro_bufr2ioda.f90

```diff
@@ -567,4 +628,36 @@ SUBROUTINE W3FS21(IDATE, NMIN)
     RETURN
 END SUBROUTINE W3FS21
 
+!-------------------------------------------------------------
+! written by H. ZHANG based w3nco_v2.0.6/w3fs21.f and iw3jdn.f
+! calculating epoch time since January 1, 1970
+SUBROUTINE epochtimecalculator(IDATE, EPOCHTIME)
+    INTEGER    IDATE(5)
```

Comment:
IDATE(5) should be IDATE(6).
IDATE(6) is used in line 660.

Reply:
Thanks for catching this, @jamiebresch! I just created a PR with these changes. Please, see https://github.com/JCSDA-internal/ioda-converters/pull/780. I'll mark this conversation as resolved though.